vivisect/qt/funcgraph.py

```diff
@@ -489,7 +489,10 @@ def _getNodeSizes(self, data):
         js = 'var sizes = {};'
 
         for nid, nprops in self.graph.getNodes():
-            cbname = 'codeblock_%.8x' % nid
+            try:
+                cbname = 'codeblock_%.8x' % nid
+            except:
+                return
```

Comment:
does this hide some underlying error? seems like this is an operation that shouldn't really fail (unless `nid` is not what we think, and there's an underlying programming error).

Reply:
So, I haven't had a chance to go back to the pyqt5 runJavaScript stuff to make it not super slow, and I've noticed that if you drag in a function, get impatient (like me) and then drag in another, it'll scream and crash because of this. I don't 

So it shouldn't be hiding anything, but adding a logging message wouldn't be amiss.